elytron/src/test/resources/org/wildfly/extension/elytron/mycredstore.cs

```diff
@@ -0,0 +1,2 @@
+# Properties Credential Store (Do Not Modify)
```

Comment:
Would be good to mention how this was generated.

Reply:
Added